gym_xiangqi/constants.py

```diff
@@ -68,7 +68,16 @@
 DEAD = 0
 ALIVE = 1
 
+# Piece Size
+P_WIDTH = 58
```

Comment:
I think spelling out `PIECE_WIDTH` and `BOARD_WIDTH` is clearer.

Reply:
fixed!